deploy/helm/block-explorer/templates/configmap-settings.yaml

```diff
@@ -21,45 +21,44 @@ data:
             "bech32PrefixValPub": "sifvaloperpub",
             "bech32PrefixConsAddr": "sifvalcons",
             "bech32PrefixConsPub": "sifvalconspub",
-            "bondDenom": "trwn",
+            "bondDenom": "rowan",
             "powerReduction": 1000000,
             "coins": [
                 {
-                    "denom": "trwn",
-                    "displayName": "TRWN",
-                    "displayNamePlural": "TRWNS",
+                    "denom": "rowan",
+                    "displayName": "ROWAN",
+                    "displayNamePlural": "ROWAN",
                     "fraction": 1000000
```

Comment:
I think these factions are wrong. These look like 1e6 to me. The precision we use is 1e8. I think these all need to be changed to `100000000`.

Reply:
Hey actually its 1e18 due to the way peggy tokens are getting minted from eth:

1 eth = 1000000000000000000 wei
1 eth = 1e18 wei

Reply:
https://eth-converter.com/